chain/jsonrpc/src/lib.rs

```diff
@@ -1207,13 +1207,14 @@ pub fn start_http(
     genesis_config: GenesisConfig,
     client_addr: Addr<ClientActor>,
     view_client_addr: Addr<ViewClientActor>,
-) {
+) -> Vec<(&'static str, actix_web::dev::Server)> {
```

Comment:
is it better to make this a struct? Or at least we can add a comment on what the return value is